### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by zolamk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
 # reviewpad-playground
```
Comment (xulamek) [inline README.md:1]: why

File: README.md [hunk 2]
```diff
@@ -1 +1,3 @@
...
+
```
Comment (xulamek) [inline README.md:2]: where

File: README.md [hunk 3]
```diff
@@ -1 +1,3 @@
...
+test
```
Comment (xulamek) [inline README.md:3]: how

File: README.md (hunk 1 above)
Comment (xulamek) [inline README.md:1]: test

Comment (xulamek) [inline README.md:1]: acgi

Comment (xulamek) [inline README.md:1]: terraform

Comment (xulamek) [inline README.md:1]: the fuck

Comment (xulamek) [inline README.md:1]: tesy

Comment (xulamek) [reply README.md:1]: test

File: README.md [hunk 4]
```diff
@@ -1 +1,5 @@
 # reviewpad-playground
```
Comment (zolamk) [inline README.md:1]: test

File: README.md (hunk 1 above)
Comment (zolamk) [reply README.md:1]: test

Comment (zolamk) [reply README.md:1]: test

Comment (zolamk) [reply README.md:1]: test

Comment (zolamk) [reply README.md:1]: test

**Pull request closed without merging**